docs/source/writing_testbenches.rst

```diff
@@ -230,6 +230,15 @@ the various actions described in :ref:`assignment-methods` can be used.
     # Freeze action
     dut.my_signal.value = Freeze()     # my_signal stays at current value until released
 
+.. warning::
+
+    Some actions do not work on all simulators:
+
+    - Release does not work on GHDL (see :issue:`3830`).
+    - Force and Release do not work on Verilator (see :issue:`3831`).
+    - Riviera's VPI implicitly releases signal when overwriting forced signal with normal deposit (see :issue:`3832`).
+    - Questa's FLI allows overwriting forced signal with normal deposit (see :issue:`3833`).
```

Comment:
I like the thoroughness, but we have to remember to delete these lines when upstream fixes the issue. We either need to make this more general like "not all simulators fully support this feature, see the issues page" and link a URL that does a search on the issues that returns these PRs. ***OR*** we can just add comments to those issues to update the documentation so we don't forget.

Reply:
I'd go with the second option, that way it's explicitly written in the documentation and then the lines can be deleted when upstream issues are resolved. I can add comments on the issues once this is merged.

Reply:
It may very well be the case that some simulators fix their behavior in newer versions even for older cocotb versions, and then our  published docs would not be accurate anymore...
We have links to issues based on labels for each supported simulator in the Simulator Support section.
Maybe it's enough to mention those links here?